templates/needrestart_blacklist.conf.j2

```diff
@@ -0,0 +1,20 @@
+# This file overrides the blacklist from needrestart.conf.
+# It is managed by Ansible.
+
+# Blacklist binaries
+push ( @{$nrconf{blacklist}}, (
```

Comment:
This will override blacklist, before we were appending

Reply:
Are you sure about that?
I am not a perl expert and got that part from a colleague but I wrote a small script to test this.

```
#!/usr/bin/perl

use Data::Dumper;

my %test = ();

$test{var} = [
  qr(^test$),
  qr(^foo$)
];

print Dumper(\%test);

my @append = ( qr(^1$) , qr(^2$) );

push ( @{$test{var}}, (
  qr(^bar$),
), @append );

print Dumper(\%test);
```

which returns

```
$VAR1 = {
          'var' => [
                     qr/^test$/,
                     qr/^foo$/
                   ]
        };
$VAR1 = {
          'var' => [
                     qr/^test$/,
                     qr/^foo$/,
                     qr/^bar$/,
                     qr/^1$/,
                     qr/^2$/
                   ]
        };
```
So I guess this actually works.
Please tell me if I'm wrong.

Reply:
Thanks for clearing that up!